cmd/loom/gateway/general_cmds.go

```diff
@@ -677,37 +678,21 @@ func newSetMaxWithdrawLimitCommand() *cobra.Command {
 			}
 
 			var name string
-			if len(args) <= 2 || strings.EqualFold(args[2], GatewayName) {
+			if len(args) == 0 || strings.EqualFold(args[0], GatewayName) {
 				name = GatewayName
-			} else if strings.EqualFold(args[2], LoomGatewayName) {
+			} else if strings.EqualFold(args[0], LoomGatewayName) {
 				name = LoomGatewayName
 			} else {
 				return errors.New("only Gateway or LoomCoin gateway is allowed to set max withdrawal limit")
```

Comment:
LOOM can also be withdrawn through the Binance Gateway, so we should be able to set limits there too.

Reply:
Right. I'll add it.